opensquirrel/passes/merger/single_qubit_gates_merger.py

```diff
@@ -0,0 +1,56 @@
+from typing import cast
+
+from opensquirrel.default_instructions import I
+from opensquirrel.ir import IR, Barrier, BlochSphereRotation, Instruction, Qubit
+from opensquirrel.passes.merger.general_merger import Merger, compose_bloch_sphere_rotations, try_name_anonymous_bloch
+
+
+class SingleQubitGatesMerger(Merger):
+    @staticmethod
```

Comment:
I see now that changing this to a static method has quite some consequences for how the passes are invoked. To keep consistent with how this is currently done with the the other passes, we will **at first** not have these as static methods and then create a ticket to investigate whether this should change. 
 
```suggestion
```